src/share/poudriere/common.sh

```diff
@@ -3951,12 +3951,14 @@ jail_start() {
 		    \( -depth 1 -name wrkdirs -prune \) -o \
 		    \( -type d -o -type f -o -type l \) \
 		    -exec chflags -fh schg {} +
+		[ -d "${tomnt:?}/boot/modules" ] &&
+		    chflags -R noschg "${tomnt:?}/boot/modules"
```

Comment:
Can you use `if; then; fi` please? `&&` can cause problems with pipefail so we try to avoid it style-wise.